### DISCUSSION TITLE
Adapting autocomplete into dbfolder plugin

### DISCUSSION THREAD

**Discussion by RafaelGB** [mgmeyers/obsidian-kanban#626] [Ideas & Feature Requests]
I have to thank you for your astonishing work with autocomplete. Will be a solid base to implement it into my plugin.

https://github.com/RafaelGB/obsidian-db-folder/issues/107#issuecomment-1229525595

I will let you know here when its finished. Hopefully, with 2 devs with the same base code maybe in the future I can contribute with your plugin

**Comment by RafaelGB** [comment]
So happy with the result =)

https://user-images.githubusercontent.com/11924043/187093190-4acf622a-98be-47a7-8278-b83e9a9492be.mov

**Comment by RafaelGB** [comment]
@mgmeyers  You inspired me to create dbfolder plugin and I am happy to sponsor you

**Comment by mgmeyers** [comment]
Hey @RafaelGB sorry for the late reply, been bogged down by school.

This is great! Glad you were able to get it working. I'll be playing around with your plugin soon!
